rts/lra/README.adoc

```diff
@@ -0,0 +1,55 @@
+## Build everything and run integration tests
```

Comment:
Shouldn't this be in the quickstarts repo?

Reply:
I have removed it.